src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private static final int TRUCK_COUNT = 3;
+
+    @Override
+    public List<Truck> get() {
+        List<Truck> listOfTrucks = new ArrayList<>();
+        for (int i = 0; i < TRUCK_COUNT; i++) {
+            listOfTrucks.add(new Truck());
```

Comment:
We probably want to create our trucks with some data, so consider adding all-args constructor for those purposes. Same about other models